lib/app/constants/bottom_navigation_items.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:gruene_app/app/constants/routes.dart';
+
+class BottomNavigationItem {
+  final String iconPath;
+  final String labelKey;
```

Comment:
:wrench: Instead of setting a label key as string here, you could change this as follows to keep type safety (and avoid needing a typecast):
```suggestion
  final String Function(BuildContext) getTitle;
```